client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/useClinicalDetails.ts

```diff
@@ -68,7 +68,7 @@ const useClinicalDetails = (parameters: useClinicalDetailsIncome): useClinicalDe
                     }
                     const initialDBClinicalDetailsToSet = {
                         ...initialDBClinicalDetails,
-                        isPregnant: clinicalDetailsByEpidemiologyNumber.isPregnant,
+                        isPregnant: clinicalDetailsByEpidemiologyNumber.isPregnant? true : false,
```

Comment:
I think you can delete ? true : false

Reply:
It's for edge case when the value is null

Reply:
you can just dp !!clinicalDetailsByEpidemiologyNumber.isPregnant or Boolean(clinicalDetailsByEpidemiologyNumber.isPregnant) to achieve boolean covnertion

Reply:
```suggestion
                        isPregnant: !!clinicalDetailsByEpidemiologyNumber.isPregnant
```

Reply:
That's fair I'll go the boolean() route